modules/boards/spec/features/support/board_page.rb

```diff
@@ -136,9 +136,13 @@ def expect_not_referencable(list_name, work_package)
     # Expect the given titled card in the list name to be present (expect=true) or not (expect=false)
     def expect_card(list_name, card_title, present: true)
       within_list(list_name) do
+        # Wait for the card loading to finish
+        expect(page).to have_no_selector(".loading-indicator--background")
         expect(page).to have_conditional_selector(present,
                                                   '[data-test-selector="op-wp-single-card--content-subject"]',
-                                                  text: card_title)
+                                                  text: card_title,
+                                                  # Wait only 3 seconds for non-presence expectation
```

Comment:
I think the comment no longer applies.

```suggestion
                                                  # Don't wait on non-presence expectation
```